packages/client/src/components/UI/UserField/UserField.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { AllHTMLAttributes, FC } from 'react'
+
+type UserFieldProps = {
+  number: number
+  author: string
+  avatar: string
+  score: number
+} & AllHTMLAttributes<HTMLDivElement>
+
+export const UserField: FC<UserFieldProps> = ({ ...props }) => {
```

Comment:
в чём смысл так делать? `{ ...props }`

Reply:
Ты декомпозируешь объект `props`, затем заново их собираешь в новую переменную `props`. Лишние операции, лучше так:
```suggestion
export const UserField: FC<UserFieldProps> = ({ number, author, score }) => {
```

Reply:
Пофиксил этот момент